src/main/java/com/chat/yourway/model/Contact.java

```diff
@@ -26,15 +23,18 @@
 @Getter
 @Setter
 @Entity
+@ToString
```

Comment:
@ToString in an entity can cause a cascade of unnecessary field loadings, ignoring FetchType.LAZY, and looping calls for bi-directional relationships. It is recommended to specify only the required fields in @ToString. 
Example: @ToString(of = {"nickname", "email", "isActive", "isPrivate", "role"})

Reply:
I used @tostring for checking problem with username value. In last commit I deleted it because of unnecessary. Should I add it again?

Reply:
Ok) No need to add again.